app/controllers/int_api/marketplaces_controller.rb

```diff
@@ -74,4 +76,9 @@ def set_access_control_headers
     headers['Access-Control-Allow-Origin'] = '*'
   end
 
+  def ab_test_onboarding(community_id:, alternative_flags:)
+    onboarding_flag = alternative_flags[rand(alternative_flags.size)]
```

Comment:
This is nicely random. Had to test 5 times on local environment before I got a marketplace which didn't have feature flag set.